bin/nic.pl

```diff
@@ -183,6 +186,17 @@
 	}
 	untie(@lines);
 }
+
+my $initializeGit = $CONFIG{'init_git'};
+$initializeGit = 0 if !$initializeGit;
+
+if($initializeGit == 1 &! $isSubproject) {
```

Comment:
What's this `&!` operator?

Reply:
Oops, right, that's not a perl operator. It worked correctly during my testing so I didn't even think twice.